features/git-ship/current_branch/on_feature_branch/without_open_changes/same_user_different_email.feature

```diff
@@ -0,0 +1,36 @@
+Feature: git ship: shipping a feature branch where the main committer is the same user
+
+  As a developer shipping a feature branch under my secondary email
+  I don't want the author field set
+  So that I don't
+  So that my coworker is given credit for their work
```

Comment:
This doesn't seem correct for the scenario. Did you mean something along the lines of:
> So that the feature branch is committed under my primary email